lighthouse-core/test/audits/dobetterweb/no-vulnerable-libraries-test.js

```diff
@@ -40,7 +39,30 @@ describe('Avoids front-end JavaScript libraries with known vulnerabilities', ()
     assert.equal(auditResult.details.items[0].highestSeverity, 'High');
     assert.equal(auditResult.details.items[0].detectedLib.type, 'link');
     assert.equal(auditResult.details.items[0].detectedLib.text, 'angular@1.1.4');
-    assert.equal(auditResult.details.items[0].detectedLib.url, 'https://snyk.io/vuln/npm:angular?lh=1.1.4&utm_source=lighthouse&utm_medium=ref&utm_campaign=audit');
+    assert.equal(
+      auditResult.details.items[0].detectedLib.url,
+      'https://snyk.io/vuln/npm:angular?lh=1.1.4&utm_source=lighthouse&utm_medium=ref&utm_campaign=audit'
+    );
+  });
+
+  it('fails when JS libraries w/ vulnerabilities are detected (in the semver range array)', () => {
+    const auditResult = NoVulnerableLibrariesAudit.audit({
+      JSLibraries: [{name: 'Bootstrap', version: '4.0.0-alpha', npmPkgName: 'bootstrap'}],
```

Comment:
maybe we can expose and mutate the `snykDatabase` somehow to make it clearer what this test is fixing?

just afraid our snapshot changes and this starts working/breaking for the wrong reasons or something

Reply:
> maybe we can expose and mutate the snykDatabase somehow to make it clearer what this test is fixing?

weirdly enough it is exposed, so could just replace it with just the subset needed to test this.

Reply:
exposed now. wdyt?

Reply:
oh, seems like overkill to add injection just for testing.

I meant the getter, but it's static so maybe not the best thing to override and replace. Could just make `getVulnerabilities` have `snykDB` passed in and test the method directly?

Reply:
Good thinkin. Done.